src/commands/moderations/MuteCommand.ts

```diff
@@ -8,115 +8,101 @@ export default class MuteCommand extends BaseCommand {
 	constructor() {
 		super("mute", "moderation");
 	}
+
 	async run(client: DiscordClient, interaction: CommandInteraction) {
-		const member = interaction.options.data[0].member as GuildMember;
-		const time = ms(interaction.options.data[1].value as string);
-		const reason = interaction.options.data[2]
-			? (interaction.options.data[2].value as string)
-			: "No Reason Provided";
+		const options = interaction.options.data;
+		const member = options[0].member as GuildMember;
+		const time = ms(options[1].value as string);
+		const reason = options[2]?.value as string || "No Reason Provided";
+
 		if (!time) {
 			const embed = new EmbedBuilder()
-				.setDescription("❌ Provide correct time for the mute")
-				.setColor("Red");
+				.setDescription("❌ Please provide a valid duration for the mute")
+				.setColor("RED");
 			await interaction.followUp({ embeds: [embed] });
 			return;
 		}
+
 		if (!client.muteRole.editable) {
 			const embed = new EmbedBuilder()
-				.setDescription(
-					"❌ I can't assign the mute role. Make sure I have higher position then the role"
-				)
-				.setColor("Red");
+				.setDescription("❌ I cannot assign the mute role. Make sure I have a higher role position than the mute role.")
+				.setColor("RED");
 			await interaction.followUp({ embeds: [embed] });
 			return;
 		}
-		if (member.permissions.has(["ManageMessages"])) {
+
+		if (member.permissions.has("MANAGE_MESSAGES")) {
 			const embed = new EmbedBuilder()
-				.setDescription("❌ I don't have right to mute a member of Staff Team")
-				.setColor("Red");
+				.setDescription("❌ I cannot mute a member of the staff team")
+				.setColor("RED");
 			await interaction.followUp({ embeds: [embed] });
 			return;
 		}
+
 		await member.timeout(time, reason);
+
 		const memberEmbed = new EmbedBuilder()
-			.setColor("Red")
+			.setColor("RED")
```

Comment:
discord.js moved from uppercase to capitalized camel case, or pascal thingie idr